pkgs/applications/science/math/gap/default.nix

```diff
@@ -23,27 +25,28 @@ let
   # packages absolutely required for gap to start
   # `*` represents the version where applicable
   requiredPackages = [
-    "GAPDoc-*"
-    "primgrp-*"
-    "SmallGrp-*"
+    "gapdoc"
+    "primgrp"
+    "smallgrp"
     "transgrp"
   ];
   # packages autoloaded by default if available
   autoloadedPackages = [
     "atlasrep"
-    "autpgrp-*"
-    "alnuth-*"
-    "crisp-*"
-    "ctbllib-*"
-    "FactInt-*"
+    "autpgrp"
+    "alnuth"
+    "crisp"
+    "ctbllib"
+    "factint"
     "fga"
-    "irredsol-*"
-    "laguna-*"
-    "polenta-*"
-    "polycyclic-*"
-    "resclasses-*"
-    "sophus-*"
-    "tomlib-*"
+    "io" # used by atlasrep to fetch data from online sources
+    "irredsol"
+    "laguna"
+    "polenta"
+    "polycyclic"
+    "resclasses"
```

Comment:
the above list of packages is directly taken from an internal source file in GAP. This does not take into account that the list is not closed under taking dependencies: I think at least the `utils` package is missing, possibly more.

Reply:
Also radiroots and autodocs seem to be missing

Reply:
Thanks a lot! I will add those packages now.